docs/core/tools/index.md

```diff
@@ -1,130 +1,94 @@
-setting up build servers or when you wish to install the tools without administrative privileges (do note the prerequisites 
-caveat above). You can find more information on the [install script reference topic](dotnet-install-script.md). If you are 
-interested in how to set up CLI on your continuous integration (CI) build server you can take a look at the 
-[CLI with CI servers](using-ci-with-cli.md) topic. 
-
-By default, the CLI will install in a side-by-side (SxS) manner. This means that multiple versions of the CLI tools 
-can coexist at any given time on a single machine. How the correct version gets used is explained in more detail in 
-the [driver](#driver) section. 
-
-### What commands come in the box?
-The following commands are installed by default:
 
-* [new](dotnet-new.md)
-* [migrate](dotnet-migrate.md)
-* [restore](dotnet-restore.md)
-* [run](dotnet-run.md)
-* [build](dotnet-build.md)
-* [test](dotnet-test.md)
-* [publish](dotnet-publish.md)
-* [pack](dotnet-pack.md)
+Either use the native installers or use the installation shell scripts:
+
+* The native installers are primarily used on developer's machines and use each supported platform's native install mechanism, for instance, DEB packages on Ubuntu or MSI bundles on Windows. These installers install and configure the environment for immediate use by the developer but require administrative privileges on the machine. You can view the installation instructions in the [.NET Core installation guide](https://aka.ms/dotnetcoregs).
+* Shell scripts are primarily used for setting up build servers or when you wish to install the tools without administrative privileges. Install scripts don't install prerequisites on the machine, which must be installed manually. For more information, see the [install script reference topic](dotnet-install-script.md). For information on how to set up CLI on your continuous integration (CI) build server, see [Using .NET Core SDK and tools in Continuous Integration (CI)](using-ci-with-cli.md).
 
-There is also a way to import more commands on a per-project basis as well as to add your own commands. This is 
-explained in greater detail in the [extensibility section](#extensibility). 
+By default, the CLI installs in a side-by-side (SxS) manner, so multiple versions of the CLI tools can coexist on a single machine. Determining which version is used on a machine where multiple versions are installed is explained in more detail in the [Driver](#driver) section.
 
-## Working with the CLI
+## CLI commands
 
-Before we go into any more details, let's see how working with the CLI looks like from a 10,000-foot view. 
-The following example utilizes several commands from the CLI standard install to initialize a new simple console application, 
-restore the dependencies, build the application and then run it. 
+The following commands are installed by default:
```

Comment:
I think we should break the commands down in basic/modification/advanced as I've started doing at https://github.com/dotnet/docs/blob/tools-index-page/docs/core/tools/index.md.